sys-boot/gnu-efi/gnu_efi-3.0.4.recipe

```diff
@@ -0,0 +1,29 @@
+SUMMARY="GNU EFI Library - Kernel support"
+DESCRIPTION="Develop EFI applications for ARM-64, ARM-32, x86_64, IA-64 (IPF), \
+and IA-32 (x86) platforms using the GNU toolchain and the EFI development \
+environment."
+HOMEPAGE="https://sourceforge.net/projects/gnu-efi/"
+COPYRIGHT="1999-2007 Hewlett-Packard Co.
+	2006-2010 Intel Co."
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="http://downloads.sf.net/gnu-efi/gnu-efi-$portVersion.tar.bz2"
+CHECKSUM_SHA256="51a00428c3ccb96db24089ed8394843c4f83cf8f42c6a4dfddb4b7c23f2bf8af"
+SOURCE_DIR="gnu-efi-$portVersion"
+
+ARCHITECTURES="any"
+DISABLE_SOURCE_PACKAGE="yes"
```

Comment:
Ah, that was probably a mistake on my part then :p the recipe before Mason's changes is what was used to generate that hpkg. Can one even create a source package without building a lib package, or such?

The "kernel" package approach isn't new; qrencode does this... there really is zero need for libs at all at this point, as there is no plans on my part to add UEFI support to kernel/userland.